charts/istio-authorizer/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 name: istio-authorizer
 description: A Helm chart for istio-authorizer
 type: application
-version: 1.17.8
+version: 1.17.9
```

Comment:
Can we use some RC version? We have some changes planned to be released tomorrow.

Reply:
fixed